components/micro-gateway-core/src/main/ballerina/src/gateway/listeners/api_gateway_listener.bal

```diff
@@ -219,10 +241,18 @@ function initiateKeyManagerConfigurations() {
     getGatewayConfInstance().setKeyManagerConf(keyManagerConf);
 }
 
-function getOauth2OutboundProvider() returns oauth2:OutboundOAuth2Provider|error {
-    oauth2:OutboundOAuth2Provider oauth2Provider = new();
-    http:ClientConfiguration clientConfig  = {
-        secureSocket : {
+public function getBasicAuthHandler() returns http:InboundAuthHandler[] {
+    // Initializes the basic auth handler
+    auth:BasicAuthConfig basicAuthConfig = {tableName: CONFIG_USER_SECTION};
+    BasicAuthProvider configBasicAuthProvider = new (basicAuthConfig);
+    http:BasicAuthHandler basicAuthHandler = new (configBasicAuthProvider);
```

Comment:
These are local variables which clearly distinguish the identifier responsibility through identifier type. So no point in using long descriptive names
```suggestion
    http:BasicAuthHandler handler = new (configBasicAuthProvider);
```